src/fmeval/eval_algorithms/summarization_accuracy.py

```diff
@@ -191,25 +186,39 @@ def evaluate(
 
         :return: A list of EvalOutput objects.
         """
-        require(
-            self.use_ray,
-            "The use_ray instance attribute of SummarizationAccuracy must be True in order "
-            "for the evaluate method to run successfully.",
+        # Create a shared resource to be used during the evaluation.
+        bertscore_shared_resource = create_shared_resource(self.bertscore_model)
+        # Create a new pipeline that uses the shared resource instead of self.bertscore_model.
+        meteor_score, rouge_score, bert_score = SummarizationAccuracy._create_transforms(
+            target_output_keys=[DatasetColumns.TARGET_OUTPUT.value.name],
+            model_output_keys=[DatasetColumns.MODEL_OUTPUT.value.name],
+            meteor_keys=[METEOR_SCORE],
+            rouge_keys=[ROUGE_SCORE],
+            bertscore_keys=[BERT_SCORE],
+            rouge_type=self.rouge_score.rouge_type,
+            use_stemmer_for_rouge=self.rouge_score.use_stemmer,
+            bertscore_model=bertscore_shared_resource,
         )
+        pipeline = TransformPipeline([meteor_score, rouge_score, bert_score])
+
         dataset_configs = get_dataset_configs(dataset_config, self.eval_name)
         eval_outputs = []
         for dataset_config in dataset_configs:
-            eval_output = util.evaluate_dataset(
```

Comment:
Note that this PR changes the signature of the `evaluate_dataset` function, hence why the parameters being passed on the left and right don't match.